src/client/dfuse/pil4dfs/hook.c

```diff
@@ -0,0 +1,981 @@
+/**
+ * (C) Copyright 2018-2021 Lei Huang.
+ * (C) Copyright 2023 Intel.
```

Comment:
Intel or Intel Corporation.?  We have a standard header here and this needs to be correct.

Reply:
"Intel Corporation". I will fix it. Thank you!